src/plugins/dashboard/server/dashboard_saved_object/migrations/migrate_to_730/migrations_730.ts

```diff
@@ -53,7 +53,7 @@ export const migrations730 = (doc: DashboardDoc700To720, { log }: SavedObjectMig
       moveFiltersToQuery(searchSource)
     );
   } catch (e) {
-    log.warning(
+    log.warn(
```

Comment:
@elastic/kibana-presentation I changed your logger to use `warn`. This is the only change.